plugins/woocommerce-blocks/assets/js/blocks/order-confirmation/additional-fields-wrapper/block.json

```diff
@@ -0,0 +1,29 @@
+{
+	"name": "woocommerce/order-confirmation-additional-fields-wrapper",
+	"version": "1.0.0",
+	"title": "Additional Checkout Fields Section",
+	"description": "Display the additional fields section.",
```

Comment:
Maybe we could be more specific about what exactly this renders, like "Displays additional checkout fields from the 'contact' and 'additional' locations" ?